check-before-deployment.md

```diff
@@ -128,6 +128,36 @@ sysctl -p
 >
 > - `sysctl -p` is to make the configuration effective without restarting the system.
 
+## Set temporary space for TiDB instances (Recommended)
```

Comment:
```suggestion
## Set a temporary space for a TiDB instances (Recommended)
```

Reply:
Two types of temporary spaces are mentioned here, and normally there're multiple TiDB instances. 

```suggestion
## Set temporary spaces for TiDB instances (Recommended)
```